pages/feedback.tsx

```diff
@@ -1,78 +1,254 @@
+                        sessionGroup.sessions.map(session => (
+                          <li key={session.Id}>
+                            <SessionInput
+                              session={session}
+                              checked={values.sessionId === session.Id}
+                              onChange={handleChange}
+                            />
+                          </li>
+                        )),
+                      )}
+                    </StyledSessionList>
+                  </details>
+                )}
+              </StyledFormRow>
+            )}
+            <StyledFormRow>
+              <StyledHeadingLabel>How would you rate the talk?</StyledHeadingLabel>
+              <StyledRatingInput
+                type="radio"
+                name="rating"
+                value="-1"
+                id="negative"
+                checked={values.rating === '-1'}
+                onChange={handleChange}
+                required
+              />
+              <label htmlFor="negative">😞</label>
+              <StyledRatingInput
+                type="radio"
+                name="rating"
+                value="0"
+                id="neutral"
+                checked={values.rating === '0'}
+                onChange={handleChange}
+                required
+              />
+              <label htmlFor="neutral">🙂</label>
+              <StyledRatingInput
+                type="radio"
+                name="rating"
+                value="1"
+                id="positive"
+                checked={values.rating === '1'}
+                onChange={handleChange}
+                required
+              />
+              <label htmlFor="positive">😃</label>
+            </StyledFormRow>
+            <StyledFormRow>
+              <StyledLabel htmlFor="input-likes">What did you like about the talk?</StyledLabel>
+              <StyledTextArea id="input-likes" name="likes" value={values.likes} onChange={handleChange} required />
+            </StyledFormRow>
+            <StyledFormRow>
+              <StyledLabel htmlFor="input-ideas">Do you have any ideas for improvement for the speaker?</StyledLabel>
+              <StyledSmall>Please be respectful of the speaker - positivity over negativity is appreciated</StyledSmall>
+              <StyledTextArea
+                id="input-ideas"
+                name="improvementIdeas"
+                value={values.improvementIdeas}
+                onChange={handleChange}
+                required
+              />
+            </StyledFormRow>
+            <StyledSubmitButton kind="primary" type="submit">
+              Submit feedback
+            </StyledSubmitButton>
+          </StyledForm>
+        )}
+      </StyledContainer>
+    </Page>
+  )
+}
+
+Feedback.getInitialProps = async ({ res, req }) => {
+  const dates = getConferenceDates(Conference, dateTimeProvider.now())
+  // DEBUG - Hope I don't check this in but I can't get here otherwise...
```

Comment:
I couldn't get to the page without this. It's ok for now but it shouldn't make it in the end

Reply:
`Doesn't the testing control allow you to do this?`

Reply:
I think you have to use a next link for it to work. the feedback page doesn't appear in the main nav. With that said, should it be in the main navigation?

Reply:
I just found the link 😔

Reply:
As it's a separate page yes it should. We should also look at all the places we already had a link to feedback and change them, including from the agenda.